proto/process/agent.proto

```diff
@@ -77,14 +77,16 @@ message CollectorConnections {
 
 	ContainerHostType containerHostType = 15;
 
-	CollectorConnectionsTelemetry telemetry = 16;
+	CollectorConnectionsTelemetry connTelemetry = 16;
```

Comment:
Would renaming this cause problems in the backend?

Reply:
No, because the name of a protobuf field is not preserved during encoding, just the field number - so regardless of what protobuf version the agents are using, when the backend decodes these payloads the name it'll use for this field will just be whatever name the backend's `.proto` file says ([source](https://developers.google.com/protocol-buffers/docs/encoding#structure))